### PR TITLE
Update firefox_privacy_notice.md

### PR DESCRIPTION
**Pull request by jejohnsonmoz**
Minor edits relating to Google Play requirements, opting out of Accuweather suggestions and clarifying marketing campaign data sharing.

### CODE REVIEW

File: en/firefox_privacy_notice.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Mozilla generally receives royalties when you use one of the default or optional
...
+* __Location data__: Firefox temporarily sends Mozilla your IP address, which we use to suggest content based on your country, state, and city. Mozilla may share location information with our [partners](https://support.mozilla.org/kb/firefox-suggest#w_who-are-mozillas-partners), but partners will not receive your IP address.
+
+In the US, Mozilla may also receive keyword location search data (such as when you search for “Boston”) and share this with our partners to provide recommended and sponsored content. Where this occurs, neither Mozilla nor our partners are able to associate the keyword search with an individual user once the search suggestion has been served.You can remove this functionality at any time by turning off sponsored suggestions - more information on how to do this is available in the relevant [Firefox Support](https://support.mozilla.org/en-US/kb/firefox-suggest?as=u&utm_source=inproduct) page.
```
Comment (peiying2) [inline en/firefox_privacy_notice.md:40]: remove `en-US` in the link

**Review by peiying2 — CHANGES_REQUESTED** [review]
One minor change request.

**Comment by jejohnsonmoz** [comment]
Hi @Francesco Lodolo ***@***.***> - thanks for checking, re: the
verbiage, the language is a little choppy but it makes sense to us and we'd
like to proceed as is.

I'll go in now and make the other changes re: the /en/ and the missing
space and wrong dash etc...

**Comment by flodolo** [comment]
> verbiage, the language is a little choppy but it makes sense to us and we'd like to proceed as is.

I think the problem is that I was trying to connect `It also includes Google.` to the sentence after, not the one before. I get the meaning now.

On a practical note, we'll need the actual dates before sending this out for translation

**Comment by jejohnsonmoz** [comment]
As in the actual date it will go live? January 17th...

**Comment by jejohnsonmoz** [comment]
Sorry sorry, you can ignore that - I was confusing it with the Subscription
Services Notice which needs to go live Jan 17th. If you need a specific
date, happy to use Jan 17th (if that's feasible) so we update two notices
on the same day. Otherwise the date can just be whenever the translations
are ready.

**Comment by flodolo** [comment]
What I mean is that we currently have a placeholder instead of a validity date in this update:

```
*Effective [insert data once known*
{: datetime="2023-[update once known]" }
```

This needs to be fixed before we ask the vendor to translate the content, because dates need to be localized.

Once the content is ready and reviewed, you will need to request translation via Jira (I think you have made requests for Mozilla accounts in the past, the process is the same).

**Review by jejohnsonmoz — COMMENTED** [review]
Date updated

**Review by flodolo — APPROVED** [review]

**Comment by jejohnsonmoz** [comment]
Jira ticket here: https://mozilla-hub.atlassian.net/browse/L10NV-165

**Review by peiying2 — APPROVED** [review]

**Pull request merged**